### PR TITLE
[WIP] Update argo-events chart for the upcoming v0.5 release (argoproj/argo-events#92)

### PR DESCRIPTION
**Pull request by ilyasotkov**
### Update argo-events Helm chart for the upcoming v0.5 (argoproj/argo-events#92)

- [ ] Apply principle of least privilege for `ClusterRole` resources
- [ ] Change gateway-controller docker image back to `argoproj`
- [x] Resolve https://github.com/argoproj/argo-helm/issues/20
- [ ] Be consistent with label use
- [ ] Something else?

### CODE REVIEW

**Comment by VaibhavPage** [comment]
@ilyasotkov v0.5 is released.

File: charts/argo-events/templates/gateway-controller-clusterrole.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```
Comment (andrewgdavis) [inline charts/argo-events/templates/gateway-controller-clusterrole.yaml]: is this applying the principle of least privilege?  it looks like the role has access to everything

Comment (ilyasotkov) [reply charts/argo-events/templates/gateway-controller-clusterrole.yaml]: @andrewgdavis obviously not :( This was just copied from the upstream.

I'd prefer to get it fixed in https://github.com/argoproj/argo-events/blob/master/hack/k8s/manifests/argo-events-cluster-roles.yaml (not sure if an issue is already filed or not), then copy it here.

Comment (VaibhavPage) [reply charts/argo-events/templates/gateway-controller-clusterrole.yaml]: The chart has been updated. Can we close this pr?

**Pull request closed without merging**